src/main/java/competition/subsystems/arm/LowerArmSegment.java

```diff
@@ -127,7 +126,7 @@ protected double getLowerLimitInDegrees() {
 
     @Override
     protected double getVoltageOffset() {
-        return 1.0 * Math.cos(getArmPositionInRadians());
+        return voltageOffsetProp.get() * Math.cos(getArmPositionInRadians());
```

Comment:
That's not really an offset if you're multiplying

Reply:
That's fair, it's eventually going into `getVoltageOffset` but agree it's not 100% clear from the name